### PR TITLE
[INV-40]- Improve User Table

### PR DESCRIPTION
**Pull request by muhammadjon8**

### CODE REVIEW

**Review by ShahobiddinIsmoilov — CHANGES_REQUESTED** [review]
You did not create migrationfiles. Please create migration files

File: tsconfig.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "outDir": "./dist",
     "baseUrl": "./src",
     "paths": {
-      "#/*": [
-        "*"
-      ]
+      "#/*": ["*"],
+      "@shared/*": ["shared/*"],
```
Comment (NegatiV-64) [inline tsconfig.json:15]: Let's use # as a import alias

File: tsconfig.json [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
+      "@accommodations/*": ["modules/accommodations/*"]
```
Comment (NegatiV-64) [inline tsconfig.json:16]: Don't think that we need this import alias

File: src/shared/constants/gender.constant.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export enum Gender {
+  MALE = 'Male',
```
Comment (NegatiV-64) [inline src/shared/constants/gender.constant.ts:2]: Use PascalCase for enum

**Review by NegatiV-64 — COMMENTED** [review]
I think that we need migrations

File: database/migrations/1731253332135-update-user-table-to-account.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (NegatiV-64) [inline database/migrations/1731253332135-update-user-table-to-account.ts:1]: Remove the commented ESLint

File: database/migrations/1731253332135-update-user-table-to-account.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class UpdateUserTable1731253332135 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('DROP TYPE IF EXISTS account_role_enum');
```
Comment (NegatiV-64) [inline database/migrations/1731253332135-update-user-table-to-account.ts:6]: Why we are dropping enum for role?

**Review by NegatiV-64 — COMMENTED** [review]
Please review my earlier comments

File: src/shared/configs/database.config.ts [hunk 6]
```diff
@@ -17,7 +17,7 @@ export const databaseConfig = (
   password: configService.get('DB_PASSWORD'),
   database: configService.get('DB_NAME'),
   entities: [
-    User,
```
Comment (NegatiV-64) [inline src/shared/configs/database.config.ts:20]: Profile is missing I guess

**Review by ShahobiddinIsmoilov — APPROVED** [review]

**Review by NegatiV-64 — APPROVED** [review]

**Pull request merged**